_meta/beat.yml

```
@@ -108,21 +105,21 @@ apm-server:
     # The default pattern excludes stacktrace frames that have a filename starting with '/webpack'
     #exclude_from_grouping: "^/webpack"
 
-    # If a source map has previously been uploaded, source mapping is automatically applied
+    # If a source map has previously been uploaded, source mapping is automatically applied.
     # to all error and transaction documents sent to the RUM endpoint.
     #source_mapping:
 
-      # Sourcemapping is enabled by default for RUM events
+      # Sourcemapping is enabled by default for RUM events.
```

```suggestion
      # Sourcemapping is enabled by default.
```

(sourcemapping is *only* for RUM events)